README.md

```diff
@@ -90,6 +90,7 @@ Install **both** of the following:
    - `PRIVATE_KEY` for your development wallet - `POLYGON_MUMBAI_RPC_URL`, `ETHEREUM_SEPOLIA_RPC_URL`, or `AVALANCHE_FUJI_RPC_URL`
    - `POLYGONSCAN_API_KEY`, `ETHERSCAN_API_KEY`, or `FUJI_SNOWTRACE_API_KEY` blockchain explore API keys depending on which network you're using
    - `COINMARKETCAP_API_KEY` (from [here](https://pro.coinmarketcap.com/))
+   - `SECOND_PRIVATE_KEY` (optional) , if you want to Accepts ownership of a Functions subscription after a transfer is requested. It should be private key of address you want to transfer ownership to.
```

Comment:
nit: to accept the transfer of ownership using `./tasks/Functions-billing/accept.js` you will need ton set this environment variable to act as the "transferee's" account.

Reply:
@virajpatva 
Am ignore previous. Am working on updating README.